src/users/dto/auto-suggest-users-dto.ts

```diff
@@ -0,0 +1,4 @@
+export class AutoSuggestUsersDto {
+  loginSubstring: string;
```

Comment:
you probably also should check there for adequate data types

Reply:
Done.